src/Models/Booking/Request/CancelBookingRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace StockportGovUK.NetStandard.Gateways.Models.Booking.Request;
+
+public class CancelBookingRequest
+{
+    public Guid BookingId { get; set; }
+    public string Reason { get; set; }
+    public Guid UserId { get; set; }
+    public DateTime DateCreated { get; set; }
```

Comment:
You don't need this DateCreated property in the request at all, you can just use DateTime.Now in the API